api/organisations/invites/views.py

```diff
@@ -42,13 +42,12 @@ def join_organisation_from_email(request, hash):  # type: ignore[no-untyped-def]
     invite = get_object_or_404(Invite, hash=hash)
     try:
         request.user.join_organisation_from_invite_email(invite)
+        register_hubspot_tracker(request)
```

Comment:
I'm not sure why this needed to be moved into the `try` / `except` here. I can't see any reason that `register_hubspot_tracker` would throw an `InvalidInviteError`?